cli/pkg/app/app.go

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"io"
+)
+
+const api string = "http://localhost:8000"
```

Comment:
should point to api.hub.tekton.dev. more importantly this should be overridable using an `ENV` or an `--api-server=https://my.hub.api.server` flag (viper? 🤔 )